src/config/logger.ts

```diff
@@ -0,0 +1,49 @@
+import { resolve } from 'path'
+import { createLogger, format, transports } from 'winston'
+
+const { combine, timestamp, label, printf } = format
+
+const myFormat = printf(({ level, message, label, timestamp }) => {
+  return `${timestamp} [${label}] ${level}: ${message}`
+})
+
+const { NODE_ENV } = process.env
+
+const getLogName = () => {
+  const DAY_MS = 24 * 60 * 60 * 1000
+  const now = Date.now()
+  const todayTimestamp = now - now % DAY_MS
+
+  return resolve(__dirname, `../../logs/${NODE_ENV}_${todayTimestamp}.log`)
```

Comment:
This path is not cross-platform, use only path API to make whole URI.
```suggestion
  return join(process.cwd(), 'logs', NODE_ENV, todayTimestamp.log)
```